tests/Doxyfile

```diff
@@ -657,6 +657,17 @@ INPUT                  = ../Bootloader/firmware/src \
                          ../common \
                          ../firmware/src \
                          ../firmware/src/system_config/ethernet_sk2/system_settings.h \
+                         doxygen/main.md \
```

Comment:
The order the files appear in controls the section ordering in the output. I haven't worried about ordering for now, once I write an intro I'll clean it up.